pallets/nft/src/lib.rs

```diff
@@ -30,10 +27,10 @@ pub type CID = Vec<u8>;
 #[repr(u8)]
 #[derive(Encode, Decode, Clone, Copy, BitFlags, RuntimeDebug, PartialEq, Eq)]
 pub enum ClassProperty {
-	/// Token can be transferred
-	Transferable = 0b00000001,
-	/// Token can be burned
-	Burnable = 0b00000010,
+    /// Token can be transferred
```

Comment:
Currently we should use tab instead of white space?

Reply:
It's too hard to track the differences

Reply:
already replaced indentation to tab